code/dqc/timediff.q

```diff
@@ -0,0 +1,12 @@
+\d .dqc
+
+/ Takes a table name tn and two column names ca and cb, as well as a percentage
+/ pt and a tolerance tl in milliseconds.
+timediff:{[tn;ca;cb;pt;tl]
+  .lg.o[`timediff;"HELLO"];
```

Comment:
For debugging?

Reply:
oops, shouldn't have let that slip through